### PR TITLE
Build data model for the Forex API

### PR DESCRIPTION
**Pull request by pakubieniec**
1. Add package DataModel and Mappers to project.
2. DataModel includes classes: 
       ExchangeRateInfo, ExchangeRateInfoDTO 
           contains: 
           variable date
           a list with all responses from the Forex API
           a method writeValueCurrencyExchangeToList() that adds answers to the ArrayList
       ExchangeRate, ExchangeRateDTO
           contains:
           variable ticker, exchangeAmount, rate;
           a method currencyConventer()
3. Mappers has been created for above classes.
4. Mappers has been tested in Main

### CODE REVIEW

File: src/main/java/DataModel/ExchangeRateInfo.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package DataModel;
+
+import services.FetchToTheForexAPI;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExchangeRateInfo {
+    FetchToTheForexAPI fetch = new FetchToTheForexAPI();
```
Comment (blazejprzyluski) [inline src/main/java/DataModel/ExchangeRateInfo.java]: Make private

File: src/main/java/DataModel/ExchangeRateInfoDTO.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package DataModel;
+
+import services.FetchToTheForexAPI;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExchangeRateInfoDTO {
+    FetchToTheForexAPI fetch = new FetchToTheForexAPI();
```
Comment (blazejprzyluski) [inline src/main/java/DataModel/ExchangeRateInfoDTO.java]: Could be private

File: src/test/java/TestMapper.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import DataModel.ExchangeRate;
+import DataModel.ExchangeRateDTO;
+import Mappers.ExchangeRateMapper;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class TestMapper {
```
Comment (blazejprzyluski) [inline src/test/java/TestMapper.java:10]: please move mapper tests here

File: src/test/java/TestExchangeRate.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.Test;
+
+public class TestExchangeRate {
+
+    ExchangeRate er = new ExchangeRate();
+    @Test
+    public void exchange100EURget420ZL(){
+        Assertions.assertEquals(420.0, er.getConvertedCurrency(100));
```
Comment (blazejprzyluski) [inline src/test/java/TestExchangeRate.java]: what happens when value is negative?

Comment (blazejprzyluski) [reply src/test/java/TestExchangeRate.java]: Also when value is zero, null

**Review by blazejprzyluski — APPROVED** [review]
Looks good to go!

**Comment by pakubieniec** [comment]
It's great :-)

**Pull request merged**